website/source/docs/post-processors/vsphere-template.html.md

```diff
@@ -60,7 +60,22 @@ Optional:
 
 -   `insecure` (boolean) - If it's true skip verification of server
     certificate. Default is false
-
+    
+-   `snapshot_enable` (boolean) - Enable snapshot creation before marking as a 
+    template. Default is false
+    
+-   `snapshot_name` (string) - Name for the snapshot. 
+    Require when `snapshot_enable` is enabled
+    
+-   `snapshot_description` (string) - Description for the snapshot. 
+    Require when `snapshot_enable` is enabled
+    
+-   `snapshot_memory` (boolean) - Enable memory option for the snapshot. 
+    Default is false
+    
+-   `snapshot_quiesce` (boolean) - Enable quiesce option for the snapshot. 
```

Comment:
Based on the [VMware docs](https://pubs.vmware.com/vi3/sdk/ReferenceGuide/vim.VirtualMachine.html):

> If TRUE and the virtual machine is powered on when the snapshot is taken, VMware Tools is used to quiesce the file system in the virtual machine. This assures that a disk snapshot represents a consistent state of the guest file systems. If the virutal machine is powered off or VMware Tools are not available, the quiesce flag is ignored.

I think that means this option is only relevant if the vm is powered on when the snapshot is taken, which is not the case when the vmware-template post-processor is run. I think it would be better not to add an option that has no meaning for users, and just always set this to `false` in the API call.

Reply:
ok, that sounds good